### PR TITLE
Upgrade/react version

### PR DESCRIPTION
**Pull request by bastiengrignon**
Linked with this issue #58

### CODE REVIEW

**Comment by xavierbriole** [comment]
Thank you for your contribution ! I appreciate :)

Have you tested it inside a React 17 project (using Create React App, for example) ?

**Comment by bastiengrignon** [comment]
> Thank you for your contribution ! I appreciate :)
> 
> Have you tested it inside a React 17 project (using Create React App, for example) ?

You're welcome 😃 
No I haven't tested it yet ! But can I, without the package published on npm ?

**Comment by xavierbriole** [comment]
@bastiengrignon Yes! You can try : `yarn add file:/path/to/local/folder`

**Comment by bastiengrignon** [comment]
> @bastiengrignon Yes! You can try : `yarn add file:/path/to/local/folder`

I tried but my project is in Typescript and it doesn't seem to work for me 😞

**Comment by bastiengrignon** [comment]
I've just tested it with `create-react-app` and it works great with react `v17.0.2`

**Comment by xavierbriole** [comment]
Indeed, a CRA with default template can import `react-cookienotice` but not a CRA with Typescript template.

A Typescript version of this module is coming.

File: package.json [hunk 1]
```diff
@@ -62,8 +63,8 @@
     "microbundle-crl": "^0.13.11",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.1.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^17.0.2",
```
Comment (xavierbriole) [inline package.json:66]: Can you please update the version in `peerDependencies` too ?

**Review by xavierbriole — CHANGES_REQUESTED** [review]
Just this little detail and we're good to go!

**Comment by xavierbriole** [comment]
As you mentioned in issue #40, I already started to work on a Typescript version that will radically change the design and behavior.

If you're interested to contribute, I can push the WIP in another branch (and open a new topic in GitHub Discussions) ;)

**Pull request merged**